src/components/loader/loader-square.style.js

```diff
@@ -47,13 +47,15 @@ const StyledLoaderSquare = styled.div`
 StyledLoaderSquare.defaultProps = {
   theme: baseTheme,
   size: 'small',
-  isInsideButton: false
+  isInsideButton: false,
+  loading: 'on'
```

Comment:
'on' and 'off' made sense as values when the prop was `state`, but `loading` probably makes more sense as a boolean? Appreciate it will mean some additional refactoring, but it looks a bit odd as it is now.